ui/main/src/app/modules/processmonitoring/processmonitoring.component.scss

```diff
@@ -77,6 +77,61 @@
     padding-left: 5%;
 }
 
+#opfab-processmonitoring-date-navigation-links {
+    position: relative;
+    float: right;
+    flex-grow: 1;
+    font-size: 13px;
+    color: var(--opfab-navbar-color);
+    background-color: var(--opfab-bgcolor);
+    text-decoration: none;
+    border-radius: 5px;
+    padding-bottom: 5px;
+    margin-top: -5px;
+    height: 2em;
+
+    ul {
+        margin: 0;
+        list-style: none;
+        display: flex;
+        justify-content: flex-end;
+
+        li {
+            padding-left: 0px;
+            padding-right: 25px;
+            // The following 3 lines avoid having the text go an a new line
+            // Arise only on chrome (OC-1563)
+            text-overflow: clip;
+            white-space: nowrap;
+            overflow: hidden;
```

Comment:
It seems it is not needed in this use case (i tested locally with ng serve mode)

Reply:
Done.